src/ckeditor.js

```diff
@@ -98,13 +98,15 @@ export default {
 
 	watch: {
 		value( val ) {
-			if ( this.instance.getData() !== val ) {
+			if ( this.instance && this.instance.getData() !== val ) {
 				this.instance.setData( val );
 			}
 		},
 
 		readOnly( val ) {
-			this.instance.setReadOnly( val );
+			if ( this.instance ) {
```

Comment:
Ok, I see there are two things - waiting for `getEditorNamespace()` - after it is done, `config.readOnly` gets the value from component `readOnly` property and then `this.instance` is initialized - this means this will work fine and value will be up to date https://github.com/ckeditor/ckeditor4-vue/blob/c19bec9976363a59d5d607a9ae5f2b862f8a882e/src/ckeditor.js#L48-L61

The other thing is time between editor start end end of initialization - `readOnly` component property may changed in the meantime but then it will be handled by `readOnly` watcher.

Reply:
So we are safe here too :+1:

Reply:
https://github.com/ckeditor/ckeditor4-vue/pull/33#discussion_r425765220